tools/internal_ci/linux/grpc_distribtests_python.sh

```diff
@@ -23,10 +23,19 @@ cd $(dirname $0)/../../..
 
 source tools/internal_ci/helper_scripts/prepare_build_linux_rc
 
-# some distribtests use a pre-registered binfmt_misc hook
-# to automatically execute foreign binaries (such as aarch64)
-# under qemu emulator.
-source tools/internal_ci/helper_scripts/prepare_qemu_rc
+IS_AARCH64_MUSL=""
+if [[ "${TASK_RUNNER_EXTRA_FILTERS}" == "aarch64 musllinux_1_1" || "${TASK_RUNNER_EXTRA_FILTERS}" == "presubmit aarch64 musllinux_1_1" ]]; then
```

Comment:
Might be cleaner to something like

```bash
if echo "${TASK_RUNNER_EXTRA_FILTERS}" | grep "aarch64" && echo "${TASK_RUNNER_EXTRA_FILTERS}" | grep "musllinux_1_1""; then 
```

Conceptually, this is kind of like the following in Python:

```python
if "aarch64" in filters and "musllinux_1_1" in filters:
```

Should be a little less brittle.

Reply:
Right, `grep` is better here, I'll change it to `grep`.